test/index.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+/* eslint-env mocha */
+/* eslint no-proto: 0 */
+var assert = require('assert')
+var setPrototypeOf = require('..')
+
+function protoTest (obj, proto) {
+  if (Object.getPrototypeOf) {
+    assert.strictEqual(Object.getPrototypeOf(obj), proto)
+  } else if ({ __proto__: [] } instanceof Array) {
+    assert.strictEqual(obj.__proto__, proto)
+  } else {
+    for (var prop in obj) {
+      if (obj.hasOwnProperty(prop)) {
```

Comment:
This line is kind of meaningless because if you manually add a field to an object then the hasOwnProperty will evaluate to true. I have provided a picture
![capture](https://user-images.githubusercontent.com/5825929/50263628-c68bfa80-03e4-11e9-9a81-30749ddec3ec.PNG)

Reply:
The whole point of this method is to reverse the functionality which the polyfill provides.  Doing the test check in a different way than the module itself seems non-intuitive to me.

Reply:
this test is more narrow. However, it is more accurate because it takes into consideration of obj original fields. Also this test will fail in certain situation. For example, when it loops through object which is { a: 1, b: 2, c: 3, d: 4} versus proto which is { c: 3, d: 4} then the a and b fields will fail because proto will return undefined while obj will return 1 and 2 respectively

Reply:
Yeah, I think you are right, this is not fully the correct test.  Technically we need to loop through both the `obj` and `proto` properties to ensure they are correct.  I will fix that, but since there is no environment to test this with in the current setup, this code is not actually being hit.

The last time I setup browser testing it was with with Sauce Labs, I will look and see if I can revive an integration with that, but I don't want to block this release on figuring it out.  I will update the test and then release, and look at the browser testing after.